docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -87,20 +87,20 @@ To remove a connection, simply click on the connection directly and then press t
 
 ### Adjust connectors
 
-You can adjust connectors, adding angles to them, to fit the canvas you're working in. When you move connected elements, the connector resizes to fit the space. To adjust a connector, click it to display the midpoint controls and move those as needed. To make a connector a straight line again, move the element or connector until the midpoint controls disappear.
+You can adjust connectors, adding angles to them, to fit the canvas you're working in. When you move connected elements, the connector resizes to fit the space. To adjust a connector, click it to display the midpoint controls and move those as needed. To make a connector a straight line again, move the midpoint back until the midpoint controls disappear.
 
-By default, if you move a connector so that it's almost a right angle or a straight line, the connector snaps into that angle or into a straight line. To disable this behavior, press and hold the Ctrl key while adjusting the connector.
+If you move a connector so that it's almost a right angle or a straight line, the connector snaps into that angle or into a straight line.
 
-{{< video-embed src="/media/docs/grafana/panels-visualizations/gif-canvas-connector-vertex-control-v11.0.mp4>" alt="Changing a connector from a straight line to a right angle" >}}
+<!-- TODO: Fix gif / use updated demo from what's new -->
```

Comment:
@nmarrs We can repurpose the short demo that Drew did here, but we'll have to dump it at some point for something more evergreen. It's gonna be weird by about 11.2 to be talking about "new features". What do you think?

Reply:
Sounds good to me :)

Reply:
Let's wait until it's been fancied up by the content team to replace this gif. In the meantime I think we can merge this PR?